File: mario.c
#include <stdio.h>
#include <cs50.h>

int main(void)

{
    int height;  //height of the pyramid

    do
    {
        height = get_int("Height:");   //promting the user for height
    }
    while (height > 8 || height < 1);

    {

        for (int i = 1; i <= height; i++)   // the for loop for height(number of rows)
        {
            for (int space = 1; space <= height - i ; space++)
            {
                printf(" ");
            }

            for (int hash = 1; hash <= i; hash++)    //inner nested for loop for printing spaces
            {
                printf("#");
            }


            printf("\n");

        }
    }


}

File: mario.py
from cs50 import get_int

while True:
    height = get_int("Height: ")

    if height <= 8 and height >= 1:
        break


for i in range(1, height + 1):
    for space in range(1, height + 1 - i):
        print(" ", end = "")

    for hash in range(i):
        print("#", end = "")

    print()


#including those +1 in the for loop, because the range function does not include
#1,height+1 means it will start with 1